File: C/D3/1 - max num/1.cpp
// Write a C program for find a max number of the 2 numbers using turnary operator.

#include <stdio.h>

int main(){
    int a, b, max;

    printf("Enter values: ");
    scanf("%d %d", &a, &b);

    max = a>b ? a : b;
    printf("Max number is %d", max);
}

File: Python/D12 - T/1/1.py
# Write a program to find the number of times an element occurs in the list.


list1 = []

n = int(input('Enter number of elements :'))
count = 0

for i in range(n):
    x = int(input('Enter element: '))
    list1.append(x)
print()

val = int(input('Enter value to find : '))
count = list1.count(val)


print(count)

